\documentclass[10pt,a4paper,twocolumn]{article}

\input{pisika.dat}

%  Editorial staff will uncomment the next line
% \input{staff.hed}


\begin{document}

%--------------------------------------------------------------------------
%  fill in the paper's title, author(s), and corresponding institutions
%--------------------------------------------------------------------------
\providecommand{\ShortAuthorList}[0]{David Beinhauer}
\title{Differences between Acetylcholinesterase Inhibitors}
\author[1,*]{David Beinhauer}
\affil[1]{Faculty of Science, Bioinformatics, Praque, Czech republic}
\affil[*]{david.beinhauer@email.cz}

\date{\dateline{}}

\begin{abstract}
\noindent
%---------------------------------------------------------------------------
%               Include abstract and keywords here
%---------------------------------------------------------------------------

The goal of this project is to compare two structures of human acetylcholinesterase
(AchE), focused on active sites. Each of the structure binds different ligand. 
The selected
ligands are inhibitors used in the treatment of the Alzheimer's disease. The idea is to
compare the active site iteractions and point out the differences between them. The
choosen structures are 4EY6 and 4EY7. The ligand in the first structures is 
galanthamine (GNT), in the second structure is dopenezil (E20). 


\DOI{} % do not delete this line
\end{abstract}

\maketitle
\thispagestyle{titlestyle}

%---------------------------------------------------------------------------
%               the main text of your paper begins here
%---------------------------------------------------------------------------


\section*{Introduction}

Alzheimer's disease (AD) is a chronic, progressive, neurodegenerative disorder of the 
brain characterized clinically by deterioration in the key symptoms of activities 
of daily living (ADLs), behavior, and cognition. Based on the 
cholinergic hypothesis\textsuperscript{\cite{bartus1982cholinergic}},
the cognitive decline in AD is a result of the deficits in central cholinergic 
neurotransmission resulting from a loss of acetylcholine (Ach).

Normally, the actions of Ach are terminated by a specific mechanism 
to keep the target cells from becoming overactivated. Acetylcholine is 
destroyed by an enzyme\textsuperscript{\cite{KOMERSOVA2005387}}, 
acetylcholinesterase (AchE), that is located in every 
Ach synapse. The defect course of enzymatic hydrolysis of 
Ach is considered to be one of the possible reasons of 
AD\textsuperscript{\cite{KOMERSOVA2005387}}. 
Cholinesterase inhibitors enhance central cholinergic 
function, and their usage remains the standard approach to the symptomatic 
treatment of AD\textsuperscript{\cite{grossberg2003cholinesterase}}.

In this work, we focus on two commonly used AChE inhibitors for the symptomatic 
treatment of AD, galanthamine and dopenezil. Mainly, we focus on the binding site
interactions of these inhibitors with human AchE, and we will point out the 
differences between these interactions.



\section*{Methods}

\subsection*{Used structures}
Two molecular structures of human AchE binded with different ligands were 
used. PDB code of protein-ligand complex AchE with galanthamine 
(PDB code: GNT) is 4EY6. PDB code of complex AchE with dopenezil 
(PDB code: E20) is 4EY7. AchE is a homodimer. Each chain contains 
binding site for the selected ligands. Both structures were crystalized
using the same approach in the study presenting several crystal 
structures of AchE in complexes with drug 
ligands\textsuperscript{\cite{doi:10.1021/jm300871x}}. Resolution of 
4EY6 is 2.4\AA, resolution of 4EY7 is 2.35\AA. Based on PDB validation
report, quality of both structures are reasonably good and in all studied
metrics result above average quality.

Besides the mentioned ligands GNT and E20, the structures contain other ligands.
The structure 4EY6 contains besides GNT also PE8, NAG, EDO and NO3 ligands. 
The structure 4EY7 contains besides E20 also NAG, EDO and NO3 ligands. Presence 
of these ligands is not mentioned in the original 
study\textsuperscript{\cite{doi:10.1021/jm300871x}}. Possible explanation of 
presence of PE8, EDO and NO3 might be that these are crystallization artifacts.
The bonded oligosaccharide NAG might be result of post-translational 
modifications of the protein. As our study is focused on the interaction of GNT 
(respectively E020) and these additional ligands do not interact with our focused
protein-ligand interaction, we do not investigate these interactions more profoundly.


\subsection*{Methods and Workflow}
Acetylcholinesterase structures were investigated using the program PyMOL. First, 
we aligned the structures to verify its similarity. Then, we hid one of the 
chains (as it is homodimer), and we showed the AchE 
surface focused mainly on the binding side where the selected
ligands bind. Furthermore, we found the polar contact between the protein and 
the ligands, and we measured the distances of the bounds. 
Finally, we filtered out the protein suroundings and focus only on the 
structure around the ligands. The finding of the polar contacts, the
measurement of the distances as well as visualizations were done 
by the appropriate functions in PyMOL.



\section*{Results}

When comparing both structures as whole, we can see almost perfect 
match between the structures (Figure \ref{fig:alignment}). Both 
structure models were created for the same study using the same 
methods, so, the high similarity is not very surprising.


\begin{figure}[tb]
    \centering
    \includegraphics[width=0.98\linewidth]{images/alignment.png}
    \caption{\textbf{Structure Comparision:} 
    Comparision of the structures 4EY6 (green) and 4EY7 (blue).}
    \label{fig:alignment}
  \end{figure} 
  

From now on, we focus only on the protein-ligand binding-site. 
First, we compare the structures based on the surfaces 
(Figure \ref{fig:surfaces}). We can see that the bindig-site 
is quite narrow and profound. Also, we observe the significant 
difference of shape and position of the ligands. The ligand GNT is 
short and circular-like shaped. On the other hand, E20 is much 
taller and its structure looks more like string. We can better see
the difference in shapes of ligands in the figure 
\ref{fig:ligands_overlap}. Also, we need to point out the slight 
misplacement of the amino acids of the protein in the binding-site.
This might be the result of the measurement error as well as different
conformation changes cause by the ligation of different ligand.

\begin{figure}[tb]
    \centering
    \includegraphics[width=0.98\linewidth]{images/surface_4ey6.png}
    \\[\smallskipamount]
    \includegraphics[width=0.98\linewidth]{images/surface_4ey7.png}
    \caption{\textbf{Binding-site Surfaces:} 
    Comparision of protein binding-site surfaces of structures 4EY6 (green) 
    ligated with GNT (red) and 4EY7 (blue) ligated with E20 (yellow).}
    \label{fig:surfaces}
\end{figure} 


\begin{figure}[tb]
    \centering
    \includegraphics[width=0.98\linewidth]{images/ligands_overlap_1.png}
    \\[\smallskipamount]
    \includegraphics[width=0.98\linewidth]{images/ligands_overlap_2.png}
    \caption{\textbf{Ligands position comparision} 
    Comparision of positions of ligands in the protein binding-site 
    surroundings of the structures 4EY6 (green) ligated with GNT (red) 
    and 4EY7 (blue) ligated with E20 (yellow).}
    \label{fig:ligands_overlap}
\end{figure} 


Next, we focus on the binding interactions of the protein-ligand 
complex. First, we investigate the structure 4EY6. The ligand GNT
is composed of 4 aromatic cycles, so, it is possible that it interacts
with the protein by stacking interactions. We observed one such
interaction between the ligand and tryptophan TRP (86). Distance of the 
aromatic cycles was measured approximately to 4.1\AA, which is 
reasonable distance for such interaction.

\begin{figure}[tb]
    \centering
    \includegraphics[width=0.92\linewidth]{images/contacts_4ey6_1_new.png}
    \\[\smallskipamount]
    \includegraphics[width=0.92\linewidth]{images/contacts_4ey6_2_new.png}
    \\[\smallskipamount]
    \includegraphics[width=0.92\linewidth]{images/contacts_4ey6_3_new.png}
    \caption{\textbf{4EY6 binding-site contacts:} 
    Display of protein binding-site non-covalent interactions 
    of structure 4EY6 (green) with ligand GNT (red).}
    \label{fig:4ey6_contacts}
\end{figure} 


Apart of these interactions, we can also see non-covalent contacts of 
sulfur from the GNT and histidine (distance 3.4\AA) on position HIS (447) 
and nitrogen from the GNT and tyrosine (distance 2.9\AA) on position TYR (337). 
We can see the visualizations
of all interactions in the figure \ref{fig:4ey6_contacts}. 

At the end, 
we should mention that there are few other aromatic amino acids 
from the protein which might also be part of the stacking 
interactions (distance of the aromatic ring centers were around 5.5\AA). 
Though, we did not visualize them because of the 
higher uncertainty of existence of these bonds. All adepts for
stacking interactions of this structure are the following 
TRP (86), TYR (124), TYR (337) and PHE (338).

In terms of the structure 4EY7, we see much more adepts of stacking 
interactions. Ligand E20 is also composed of 4 aromatic rings and interacts by 
stacking interactions with tryptophans and tyrosines of the protein
(the distances are around 34\AA). The adepts for stacking interactions
are the following amino acids of the AchE protein 
TRP (86), TRP (286), TYR (337) and TYR (341).
Apart from this interactions, 
it also interacts with nitrogen in peptide backbone (PHE (295)) 
in distance 2.9\AA. We can see the visualizations
of all interactions in the figure \ref{fig:4ey7_contacts}. 


\begin{figure}[tb]
    \centering
    \includegraphics[width=0.92\linewidth]{images/contacts_4ey7_1_new.png}
    \\[\smallskipamount]
    \includegraphics[width=0.92\linewidth]{images/contacts_4ey7_2_new.png}
    \\[\smallskipamount]
    \includegraphics[width=0.92\linewidth]{images/contacts_4ey7_3_new.png}
    \caption{\textbf{4EY7 binding-site contacts:} 
    Display of protein binding-site non-covalent interactions 
    of structure 4EY7 (blue) with ligand GNT (yellow).}
    \label{fig:4ey7_contacts}
\end{figure} 
  

To sum up, both ligands are surprisingly very distinct in the shape.
We cannot state for sure, but it looks that both ligands interact
with the AchE protein mainly by the stacking interactions of its
aromatic rings (mainly the same rings). 
Despite, the difference of the ligands they both 
serves the same function, which is inhibition of the AchE resulting
in delay of Ach hydrolysis. 


% Please use pisikabst.bst. You may your own *.bib file.
\clearpage
\bibliographystyle{pisikabst}
\bibliography{bibfile}


\end{document}